src/main/java/com/laborete/LaboreteAPI/posts/controller/PostsController.java

```diff
@@ -47,5 +47,10 @@ private ResponseEntity<PostDTO> getPostById(@PathVariable UUID id) {
         return new ResponseEntity<>(this.postsService.getPostById(id), HttpStatus.OK);
     }
 
+    @ApiOperation("Get posts filtered by searchValue")
+    @PostMapping({"/filter"})
+    private List<PostDTO> findByTextLike(@RequestBody String searchValue) {
```

Comment:
In the future the endpoint would provide more functionality (such as ordering posts by fields). So, the method should take a special dto as an argument. Please create a `FilterDto`. At the moment this dto is gonna contain `searchValue` field only, but in the future we would add some more extra fields (like `order` field that specifies fields to order the posts by). Then, we can also rename the method itself - `filterPosts` sounds good for me.